base/applications/mscutils/eventvwr/lang/ja-JP.rc

```diff
@@ -123,13 +123,13 @@ BEGIN
     EDITTEXT IDC_LOGNAME, 67, 23, 178, 12, ES_LEFT | ES_AUTOHSCROLL | ES_READONLY | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
     LTEXT "ログファイル(&F):", IDC_STATIC, 7, 41, 60, 8
     EDITTEXT IDC_LOGFILE, 67, 39, 178, 12, ES_LEFT | ES_AUTOHSCROLL | ES_READONLY | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
-    LTEXT "サイズ(&S):", IDC_STATIC, 7, 57, 60, 8
+    LTEXT "サイズ:", IDC_STATIC, 7, 57, 60, 8
     EDITTEXT IDC_SIZE_LABEL, 67, 57, 178, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "作成日時(&C):", IDC_STATIC, 7, 69, 60, 8
+    LTEXT "作成日時:", IDC_STATIC, 7, 69, 60, 8
     EDITTEXT IDC_CREATED_LABEL, 67, 69, 178, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "変更日時(&M):", IDC_STATIC, 7, 81, 60, 8
```

Comment:
Ditto

Reply:
see above.

Reply:
resolved.

Reply:
For the record: You even had 2 colliding "(&M)" in that dialog.